docs/Android/privacy-config.md

```diff
@@ -0,0 +1,212 @@
+|                     |                                                                                     |      AVAILABLE AND EMPTY     | All attributes are computed and reported for the corresponding entity.                                |                                                                                                                                                        |
+|                     |                                                                                     |    AVAILABLE AND NOT EMPTY   | Attributes in the array are disabled from being computed or transmitted for the corresponding entity. |                                                                                                                                                        |
+|   EventsToDisable   |                 Used for disabling specific events within an entity                 |                              |                                                                                                       |                                                                                                                                                        |
+|                     |                                                                                     |            MISSING           | All events are computed and reported for the corresponding entity.                                    |                                                                                                                                                        |
+|                     |                                                                                     |      AVAILABLE AND EMPTY     | All events are computed and reported for the corresponding entity.                                    |                                                                                                                                                        |
+|                     |                                                                                     |    AVAILABLE AND NOT EMPTY   | Events in the array are disabled from being computed or transmitted for the corresponding entity.     |                                                                                                                                                        |
+
+
+### Examples
+
+#### All enabled
+To have all Telemetry Feature data enabled, no configuration payload needs to be injected through the SDK. All that is needed is to enable the Telemetry Feature itself.
+
+However, if a payload needs to be specified it can be structured in the following way:
+```JSON
+{
+  "DEXData": {
+    "Version": 1.0
+    // No category data blocks as defaulted to enabled
```

Comment:
This JSON block and the below are erroring out as well. Could the comments be added outside the blocks?